documentation/docs/content-and-navigation/demos/skip-links.tsx

```diff
@@ -67,7 +67,7 @@ export const DemoSkipLinks = () => {
 						<a
 							className={styles.skipLinks__anchor}
 							id="website"
-							href="https://github.com/feedzai/react-a11y-tools"
+							href="https://github.com/feedzai/react-a11y-tools/"
```

Comment:
Since this is already a big MR if some changes can be ignored (such as adding a extra '/') i would not add it (if not necessary ofc). There are already some files that only add a space and if those changes are not strictly necessary we could revert it